lib/rake_task_helpers/staging_sync.rb

```diff
@@ -1,3 +1,8 @@
+# This task is mainly for trimming down size of DB and anonymising the users' information
+# for non-staging env. Not sure why its name contains 'Sync'? which cause confusing as
+# there is no step of cloning live db in this part.
+# Right now it is designed to run on daily baisis which I couldn't see the point
```

Comment:
I agree I think it could run over a longer period and that would be fine. I think we should amend this comment though when this is changed as it could become confusing in future as it refers to the current point in time but this will become out of data as a statement as things change.

Reply:
I think I just remove those comments.